diff --git a/tensorflow/core/distributed_runtime/BUILD b/tensorflow/core/distributed_runtime/BUILD
--- a/tensorflow/core/distributed_runtime/BUILD
+++ b/tensorflow/core/distributed_runtime/BUILD
@@ -31,7 +31,7 @@ load(
 )
 
 package(default_visibility = [
-    "//tensorflow:internal",
+    "//visibility:public",
 ])
 
 cc_library(
diff --git a/tensorflow/core/distributed_runtime/rpc/BUILD b/tensorflow/core/distributed_runtime/rpc/BUILD
--- a/tensorflow/core/distributed_runtime/rpc/BUILD
+++ b/tensorflow/core/distributed_runtime/rpc/BUILD
@@ -33,7 +33,7 @@ load(
 )
 
 package(default_visibility = [
-    "//tensorflow:internal",
+    "//visibility:public",
 ])
 
 cc_library(


diff --git a/tensorflow/core/BUILD b/tensorflow/core/BUILD
--- a/tensorflow/core/BUILD
+++ b/tensorflow/core/BUILD
@@ -2060,7 +2060,7 @@
     cc_api_version = 2,
     protodeps = tf_additional_all_protos() + [],
     visibility = [
-        "//tensorflow:internal",
+        "//visibility:public",
     ],
 )

